opentreemap/treemap/js/src/mapPage/browseTreesMode.js

```diff
@@ -49,11 +52,19 @@ function init(options) {
                 .filter(inMyMode),
 
         popupHtmlStream = utfEventStream.flatMap(getPopupContent),
+
         clickedIdStream = utfEventStream.map('.data.' + config.utfGrid.mapfeatureIdKey),
-        accordionHtmlStream = BU.fetchFromIdStream(clickedIdStream,
+        pagingIdStream = $('body').asEventStream('click', dom.popupPagingButtons)
+            .doAction(changePopupSelection)
+            .doAction(toggleAccordion, '')
+            .debounce(300)
+            .map(getIdFromPopup),
+        idStream = Bacon.mergeAll(clickedIdStream, pagingIdStream),
+
+        accordionHtmlStream = BU.fetchFromIdStream(idStream,
```

Comment:
Good catch! This was caused by the `etag` header not including the list of nearby features into the hash (which I didn't notice because I had disabled my browser cache).

Fixed in https://github.com/OpenTreeMap/otm-core/pull/3181/commits/41be44fc321693a61be1b6ae56aeba53f87b92da